cli_tools/gce_image_publish/publish/publish_test.go

```diff
@@ -518,13 +682,76 @@ func TestDeprecatePrintOut(t *testing.T) {
 			p := &Publish{}
 			p.deprecatePrintOut(tt.args)
 			if !reflect.DeepEqual(p.toDeprecate, tt.toDeprecate) {
-				t.Errorf("deprecatePrintOut() got = %v, want %v", p.toDeprecate, tt.toDeprecate)
+				t.Errorf("deprecatePrintOut() toDeprecate got = %v, want %v", p.toDeprecate, tt.toDeprecate)
 			}
 			if !reflect.DeepEqual(p.toObsolete, tt.toObsolete) {
-				t.Errorf("deprecatePrintOut() got1 = %v, want %v", p.toObsolete, tt.toObsolete)
+				t.Errorf("deprecatePrintOut() toObsolete got = %v, want %v", p.toObsolete, tt.toObsolete)
 			}
 			if !reflect.DeepEqual(p.toUndeprecate, tt.toUndeprecate) {
-				t.Errorf("deprecatePrintOut() got2 = %v, want %v", p.toUndeprecate, tt.toUndeprecate)
+				t.Errorf("deprecatePrintOut() toUndeprecate got = %v, want %v", p.toUndeprecate, tt.toUndeprecate)
+			}
+		})
+	}
+}
+
+func TestCreateRollOut(t *testing.T) {
```

Comment:
This doesn't test for sorting zones since test data is already sorted. Not sure that's a strict requirement though so it might not matter.

Reply:
I updated the testdata so it's no longer sorted.